aiidalab_widgets_base/static/styles/README.md

```diff
@@ -0,0 +1,15 @@
+# Stylesheets for AiiDAlab Widgets Base
+
+This folder contains the following:
+
+- `scss` - a folder for SCSS-format stylesheets to be compiled
+- `.css` files  - compiled CSS stylesheets
+
+It is recommended to add SCSS stylesheets in the SCSS folder for functionality and flexibility (see https://sass-lang.com/).
+
+Compiling SCSS into CSS can be done using:
+
+- The [compile-hero](https://marketplace.visualstudio.com/items?itemName=Wscats.eno) extension, if using VS Code
+- The `sass` [CLI](https://sass-lang.com/install/)
```

Comment:
I think we should have a simple bash script that will invoke the correct invocation of the compiler. Otherwise it's not immediately clear what needs to be done.  (this could then be used as a pre-commit hook, as noted above).

Reply:
See my last comment